data2rdf/csv_parser.py

```diff
@@ -1,12 +1,14 @@
 import io
-import uuid
 
 import magic
 import pandas as pd
 
+from data2rdf.parser import DataParser
 
-class CSVParser:
+# import uuid
```

Comment:
Instead of commenting out the code, you can remove it. We could go back to it via visiting an older commit.

Reply:
Yes, the unittests are running successfully

Reply:
Thanks